cmd/integration-test/integration-test.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/logrusorgru/aurora"
+)
+
+type TestCase interface {
+	// Execute executes a test case and returns any errors if occurred
+	Execute() error
+}
+
+var (
+	// debug      = os.Getenv("DEBUG") == "true"
+	customTest = os.Getenv("TEST")
+
+	errored = false
+	success = aurora.Green("[✓]").String()
+	failed  = aurora.Red("[✘]").String()
+
+	tests = map[string]map[string]TestCase{
+		"code": libraryTestcases,
+	}
+)
+
```

Comment:
@RamanaReddy0M  we should  also add integration test to cdncheck similar to nuclei . i.e testcases by using flags and executing commands . 

## TODO:
add test cases to check following functionalities for display/match/filter using command execution / flags
-  cdn
-  waf
- cloud

Reply:
@tarunKoyalwar we already have a functional test for cdn, waf and cloud as command execution / flags at
https://github.com/projectdiscovery/cdncheck/blob/issue-66-cdn-release-preparation/internal/testutils/testutils.go